### PR TITLE
CloudWatch: Add RDS dimension

### PR DESCRIPTION
**Pull request by jangaraj**
`EndpointName` RDS dimension is missing. Doc: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/rds-proxy.monitoring.html

### CODE REVIEW

**Review by idastambuk — APPROVED** [review]

**Comment by idastambuk** [comment]
@jangaraj Sorry for my rogue commit, rebasing with my own commit didn't succeed in passing the token to the enterprise check. 
However, I successfully built the enterprise repo locally with your change, so this commit should be ok. I will merge it once all other checks pass, namely the ones related to labels. Let me know if you need more info. 

Thank you!

**Comment by idastambuk** [comment]
@jangaraj Before I merge this, can you add a milestone, and changelog and backport labels? Thanks!

**Comment by jangaraj** [comment]
@idastambuk I would say that Grafana guys can do that. I'm external contributor, so I don't have an idea what should I put there. For example @sunker did this in my other PRs.

**Pull request merged**
